custom_model_runner/CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-#### [1.10.16] - 2023-12-08
+#### [1.10.17] - 2023-01-01
+##### Changed
+- scrub sensitive values from outputs during fit and predict
+
+#### [1.10.17] - 2023-12-08
```

Comment:
is this one suppose to be 1.10.16?

Reply:
:sigh: thank you!

`git grep --name-only '1\.10\.16" | xargs sed -i ... `  swept that up in the changes